Lombiq.Gulp.Extensions.csproj

```diff
@@ -17,6 +17,19 @@
     <InitCommonNodeModulesStampFile>..\..\node_modules\.install-stamp</InitCommonNodeModulesStampFile>
     <ESLintConfigFileTargetPath>../../.eslintrc</ESLintConfigFileTargetPath>
   </PropertyGroup>
+  
+  <Target Name="SelectPnpmVersion" BeforeTargets="InitCommonNodeModules">
+    <!-- Ignore sporadic errors when this is run by multiple threads at the same time. -->
+    <Exec Command="corepack enable"
+          IgnoreExitCode="true"
+          IgnoreStandardErrorWarningFormat="true"
+          StandardOutputImportance="$(NxVerbosity)" />
+    <!-- Enable the latest version of pnpm@6. -->
+    <Exec Command="corepack prepare pnpm@latest-7 --activate"
+          IgnoreExitCode="true"
+          IgnoreStandardErrorWarningFormat="true"
+          StandardOutputImportance="$(NxVerbosity)" />
```

Comment:
I will remove some of the safety nets here that I've put in place specifically for NE.